conf/modules.config

```diff
@@ -381,28 +371,27 @@ process {
             pattern: "*cram"
         ]
     }
-    withName: 'NFCORE_SAREK:SAREK:RECALIBRATE:MERGE_INDEX_CRAM:INDEX_CRAM' {
+    withName: 'NFCORE_SAREK:SAREK:(RECALIBRATE|RECALIBRATE_SPARK):MERGE_INDEX_CRAM:INDEX_CRAM' {
         publishDir       = [
             enabled: true,
             mode: params.publish_dir_mode,
             path: { "${params.outdir}/preprocessing/${meta.id}/recalibrated" },
             pattern: "*{recal.cram,recal.cram.crai}"
         ]
     }
-}
 
 // QC
-process{
```

Comment:
and one large process block from now on?

Reply:
absolutely. I am still votin for splitting the massive config up (in the future). Somehow we went from a monolithic pipeline -> modular pipeline but now have a massive monolithic config 😆  there is room to split it

Reply:
:exploding_head: 
what about having `modules.config` including `subworkflows/{local,nf-core}/*.config`?
or is this way too convoluted and we want to avoid that?